aries_cloudagent/connections/models/diddoc/schemas/serviceschema.py

```diff
@@ -0,0 +1,29 @@
+from marshmallow import Schema, fields, post_load
+
+
+class ServiceSchema(Schema):
+    """
+    Example
+    {"id": "1",
+     "type": "one",
+     "priority": 1,
+
+     "recipientKeys": [
+         "~XXXXXXXXXXXXXXXX",
+         "did:sov:LjgpST2rjsoxYegQDRm7EL#keys-1"],
+     "routingKeys": ["did:sov:LjgpST2rjsoxYegQDRm7EL#keys-4"],
+     "serviceEndpoint": "LjgpST2rjsoxYegQDRm7EL;2"}
+    """
+    id = fields.Str()
+    type = fields.Str()
+    priority = fields.Int()
+    recipientKeys = fields.List(fields.Str())
+    routingKeys = fields.List(fields.Str())
```

Comment:
We should have a more specific field for DID URLs. Consider using the Regex and DIDUrl class we've written in `aries_cloudagent.resolver.did` to male such a field.